gramps/gen/config.py

```diff
@@ -133,7 +133,7 @@ def emit(key):
 register('behavior.addmedia-relative-path', False)
 register('behavior.autoload', False)
 register('behavior.avg-generation-gap', 20)
-register('behavior.betawarn', False)
+register('behavior.betawarn', tuple())
```

Comment:
This should be a simple flag to indicate that the warning has been read.  Does it need to be a tuple?  Is there some problem caused by upgrading?